### ISSUE TITLE
HTTP errors

### ISSUE THREAD

**Issue by GithubTyrant** [bashrc/freedombone#74]
Don't need to explain my username I think.

To come back to the daemon (uWSGI and Nginx) not working, I think that's not the problem because of all the apps I can get a onion address I get similar errors (403 Forbidden, 500 Internal Server Error, 502 Bad Gateway)

ssh                       freedombone.local         fiwamylfi2v7hobh.onion
Email                     freedombone.local         io3hu65gb462dbyr.onion
dokuwiki                  -                         lqblnrndkxxpxqlr.onion
htmly                     -                         m4rr3hfltpdpsjse.onion
lychee                    -                         usf4tvbrdwgroltw.onion
mailpile                  -                         nz2r5dj7rkgrlfei.onion
searx                     -                         jbw7t64yiczdmt4a.onion

searx only matters to me anyways, so if you can fix that first, awesome.

other apps don't work (not suited for ARM64 I think)

This post/account will probably be deleted/disabled in a few hours/days, but with your fast response times that won't be a problem.

**Comment by bashrc** [comment]
I've tried installing these on an Odroid C2, which is aarch64 and running the Stretch version of Armbian, and the onion addresses appear to work as expected.

It could be that your OS isn't based upon Debian Stretch (9.x), which would cause chaos, or if you're running Raspbian I've not tested with that so perhaps it could differ from Debian 9 in some way.

Other things to try:

Go to the administrator control panel and enable logging.

    systemctl restart nginx
    systemctl restart php7.0-fpm
    systemctl restart tor

Try navigating to one of your sites and then check the logs. If you can report any results here then so much the better.

**Comment by GithubTyrant** [comment]
Yep that's definitely the problem, I though I was good when I update all the packages to stretch but unfortunately that doesn't work. The Orange PiPC2 only has build tools for ubuntu xenial.

That's really bad and I think I my only option is to convince the Armbian devs to make a stretch port but since this board is in development and isn't officially supported and is in beta phase I doubt it.

Any other options you can think of?

**Comment by bashrc** [comment]
I see what you mean. https://dl.armbian.com/orangepipcplus

Even though Ubuntu is debian based it's probably sufficiently divergent that all sorts of things could go wrong. A guess is that perhaps Ubuntu doesn't fully support aarch64 yet.

Looking at the u-boot-tools debian package although there are various OrangePi device trees it doesn't look like there is a PC2 one, so this must be quite a recent board. To get Debian 9 running on this board would likely be a project in itself. If I were doing that I would try to find the dtb for the debian 8 version and see if I could use that. It would mean getting into the build details and likely using debootstrap or vmdebootstrap.

**Comment by GithubTyrant** [comment]
https://dl.armbian.com/orangepipc2/

If I change the xenial parts to stretch in https://github.com/armbian/build/blob/f46b85517da0a5968f59a7a9f5521ed901995f9d/config/boards/orangepipc2.conf

Will that work?

**Comment by bashrc** [comment]
I've never built Armbian, but you could try it.
